app/controllers/application_controller.rb

```diff
@@ -164,8 +158,7 @@ def set_sentry_context
   end
 
   def switch_locale(&action)
-    locale = available_locale(params[:new_locale]) ||
-      available_locale(params[:locale]) ||
+    locale = available_locale(params[:locale]) ||
```

Comment:
replacing the new_locale param approach for switching locale with just setting the locale param in the path so this is no longer needed